src/libs/actions/BankAccounts.js

```diff
@@ -266,7 +266,6 @@ function openReimbursementAccountPage(stepToOpen, subStep, localCurrentStep) {
                 onyxMethod: CONST.ONYX.METHOD.MERGE,
                 key: ONYXKEYS.REIMBURSEMENT_ACCOUNT,
                 value: {
-                    errors: null,
```

Comment:
We re-load the `reimbursementAccount` data each time we enter the "Connect bank account" page of a workspace. If there is an error, `errors: null,` was clearing the errors, not allowing the user to clear it with the X and not giving time to read it.